Cargo.toml

```diff
@@ -45,7 +45,7 @@ kv_unstable = []
 kv_unstable_sval = ["kv_unstable", "sval/fmt"]
 
 [dependencies]
-cfg-if = "0.1.2"
+cfg-if = "=0.1.9"
```

Comment:
This will cause issues if log is used in a dependency graph with another crate with a non `=` dependency on cfg-if. Cargo will iirc just refuse to resolve versions. If we can't just `cargo update -p cfg-if --precise 0.1.9` in the MSRV build, it may just be best to bump our MSRV to whatever cfg-if uses.

Reply:
Ah right. I’d assumed we’d possibly just end up compiling two versions of `cfg-if`.

I’ll look again at why the `cargo update` path didn’t work out for me and fall back to bumping back to `1.31`.